drafts/inlinemacros.md

```diff
@@ -80,39 +86,37 @@ by-value parameters and that are not side-effect free paths. Arguments to by-nam
 parameters are left alone.
```

Comment:
I'm somewhat concerned by the fact: 1) that not only people will mistakenly write by-value parameters for inline macros and expect their trees, but 2) also that in some cases that will work. Example:

```
inline def foo(x: Int) = macro { ... x ... }
foo(42) // (1) stable, getting q"42" 
foo(M.x) // (2) stable, getting q"M.x"
foo(new C().x) // (3) not stable, getting q"x$152"
```

If a macro newbie tried to write a shiny new inline macro and only tested the first two cases, they won't even have a clue that their macro will fail on (3) and on virtually anything else.